core/src/main/java/tc/oc/pgm/api/map/Gamemode.java

```diff
@@ -20,6 +20,7 @@ public enum Gamemode {
   RACE_FOR_WOOL("rfw"),
   SCOREBOX("scorebox"),
   SKYWARS("skywars"),
+  SURVIVAL_GAMES("survival"),
```

Comment:
> I used `survival` to keep it in line with the aforementioned include and the [ProjectAres source](https://github.com/OvercastNetwork/ProjectAres/blob/7484cb81fcc33814fe26812684660da4e0dbd671/API/api/src/main/java/tc/oc/api/docs/virtual/MapDoc.java#L46), but perhaps it could be simplified to `sg`, similar to how the [Payload tag](https://github.com/PGMDev/PGM/blob/a27fc69a84c44483e670c4077bf239f39fe15dc3/core/src/main/java/tc/oc/pgm/api/map/Gamemode.java#L18) is `pd`. Do let me know and I can make the change.

Regarding Payload -> `pd` we decided `payload` was better (https://github.com/PGMDev/PGM/pull/1091#discussion_r1014908928)

Buuuuut..... I think the internal id of survival games should be `sg` and not `survival` since the abbreviation works on line with ctw/dtm etc.
```suggestion
  SURVIVAL_GAMES("sg"),
```